stable/database/templates/_helpers.tpl

```diff
@@ -84,6 +87,22 @@ Also, we can't use a single if because lazy evaluation is not an option
     {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
 {{- end -}}
 {{- end -}}
+{{- end -}}
+
+{{/*
+Renders the freeze mode for backup hooks.
+*/}}
+{{- define "backupHooks.freezeMode" -}}
+{{- if eq .Values.database.backupHooks.freezeMode "" -}}
+  {{- if (eq (include "defaultfalse" .Values.database.backupHooks.useSuspend) "true") -}}
+suspend
+  {{- else -}}
+hotsnap
```

Comment:
How is this in terms of backwards compatibility? If someone upgrades from 3.8.0 to 3.9.0 who was using fsfreeze with some nuodb version that does not support hotsnap, what happens? Do they have to explicitly configure `freezeMode=fsfreeze`?

Reply:
> Do they have to explicitly configure freezeMode=fsfreeze?
Yes, they do. Otherwise, the backup hook container will fail saying that `hotsnap` is not supported. 

It is a change in behaviour indeed but I can't think of a good way to enable `hotsnap` by default without unreliably parsing the image tag to determine if hotsnap is actually supported by the selected image.